doc/ssl/SSL_CTX_set_tlsext_status_cb.pod

```diff
@@ -16,8 +21,10 @@ Certificate Status Request functions
  long SSL_CTX_set_tlsext_status_arg(SSL_CTX *ctx, void *arg);
 
  long SSL_CTX_set_tlsext_status_type(SSL_CTX *ctx, int type);
+ long SSL_CTX_get_tlsext_status_type(SSL_CTX *ctx);
 
  long SSL_set_tlsext_status_type(SSL *s, int type);
+ long SSL_get_tlsext_status_type(SSL *s);
```

Comment:
And ok, I see that it internally happens to go through the SSL_ctrl which uses long, but
a) Why does ctrl need to support getters in the first place? Is there a good answer?
b) Even if for whatever mysterious reason we need CTRL support as well, the getter here could simply return `s->tlsext_status_type`.

There is no need to dig the API hole deeper than it already is.